starter/js/atm.js

```diff
@@ -1,12 +1,44 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
+$( document ).ready(function() {
+    var availBalChecking = 0;
+    var availBalSavings = 0;
+    var transactionAmount = 0;
+    $('#depositChecking').click(function() {
+      transactionAmount = parseFloat($('#amountChecking').val());
+      availBalChecking += transactionAmount;
+      $('#checkingBalance').html("$" + availBalChecking);
+      });
+    $('#withdrawChecking').click(function() {
+      transactionAmount = parseFloat($('#amountChecking').val());
+      if(availBalChecking - transactionAmount < 0){
+        overDraftProtection();
+        // $('#checkingBalance').html("$" + availBalChecking);
+        }
+      else{
+        availBalChecking -= transactionAmount;
+        }
+      $('#checkingBalance').html("$" + availBalChecking);
+      });
+    $('#depositSavings').click(function() {
+      transactionAmount = parseFloat($('#amountSavings').val());
+      availBalSavings += transactionAmount;
+      $('#savingsBalance').html("$" + availBalSavings);
+     });
+    function getTransactionAmount() { //for drying code
+      transactionAmount = parseFloat($('#amountChecking').val());
+    }
+   function overDraftProtection() {
```

Comment:
Nice job working on the bonus!!

You'll want a conditional to conquer this user story: "If the withdrawal amount is more than the combined account balance, ignore it". Something like `if (transactionAmount <= availBalSavings + availBalChecking)` would probably do the trick!